src/test/ceph_objectstore_tool.py

```diff
@@ -1796,16 +1797,12 @@ def main(argv):
         vstart(new=False)
         wait_for_health()
 
-        time.sleep(20)
```

Comment:
I had the exact same remark. But as long the wait_for_health is complete it should be ok. If not, then we shall make a specific test instead of waiting an amount of time which doesn't guarantee anything.
As far as we cannot determine the meaning of such wait, my proposal here is to remove it and add a real test if someone notices a regression. I didn't had any during my tests.